### DISCUSSION TITLE
Smoke Pattern Keybind

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25814] [Ideas] [answered]
I think it would be really cool if we could make something like a shape with smoke and then attach that shape to a key that will put the shape in game when the key is pressed. The idea is that there is an option where you can draw smoke on a little panel similar to the one for the tablet drivers. Then, you can choose how big the area covered by the smoke is, and where that area is placed.

- Make a panel similar to the tablet drivers, but on this panel you can draw something with smoke
- After you've drawn your shape, you can decide the size of said shape and where it will be placed on the screen
- You can then attach this shape to a keybind to use whenever you want in gameplay

I just like the idea of being able to paste a little smiley in the corner that will be the same every time, kind of like a signature. I don't know if it would be 100% perfect to implement in this way, but I see it as a good start and don't think it would be too hard, really.

**Comment by peppy** [comment, marked as answer]
Not going to happen.

**Reply by WalmartWhiteCat** [reply]
Aight.
